### PR TITLE
fix(#1286): Removed border from AppHeader items

### PR DESCRIPTION
**Pull request by ArakTaiRoth**

### CODE REVIEW

File: libs/web-components/src/components/app-header/AppHeader.svelte [hunk 1]
```diff
@@ -266,9 +266,8 @@
     display: block;
     padding: calc((3rem - var(--goa-line-height-3)) / 2) 1rem;
     text-decoration: none;
-    cursor: pointer;    
+    cursor: pointer;
     white-space: nowrap;
-    box-shadow: inset 0 var(--goa-border-width-s) 0 0 var(--goa-color-greyscale-200);
```
[the remark this answers is not in the document]
Comment (ArakTaiRoth) [reply libs/web-components/src/components/app-header/AppHeader.svelte:271]: @chrisolsen This should be resolved now, I added it as mobile only styling

[the remark this answers is not in the document]
Comment (ArakTaiRoth) [reply libs/web-components/src/components/app-header/AppHeader.svelte:271]: @chrisolsen dammit, sorry, didn't realize that we spawned the menu for tablet sizes as well, that's been resolved to

**Comment by Radha-Alberta** [comment]
- [ ] Padding issues in App Header Menu dropdown

- **Device Details**:
image


**Current Scenario:**
image

**Review by chrisolsen — APPROVED** [review]

**Pull request merged**